contrib/planb-objsync.py

```diff
@@ -1780,26 +2048,27 @@ def execute(self):
             raise NotImplementedError()
 
     def sync_container(self, container_name):
-        swiftsync = SwiftSync(self.config, container_name)
-        swiftsync.sync()
+        planbsync = PlanbSync(self.client, self.config, container_name)
```

Comment:
`PlanbObjsync` or `ObjSync` ?

Reply:
I I'm going with `ObjectSync` as there are a number of Object classes.